src/tools/clu_funcs.c

```diff
@@ -1154,6 +1154,80 @@ static void wolfCLU_AddNameEntry(WOLFSSL_X509_NAME* name, int type, int nid,
 }
 
 
+/* Input 'n' is a terminated string in the form of '/CN=name/C=company'
+ * returns a newly created WOLFSSL_X509_NAME on success */
+WOLFSSL_X509_NAME* wolfCLU_ParseX509NameString(char* n, int nSz)
+{
+    WOLFSSL_X509_NAME* ret = NULL;
+    char* word, *end;
+    char* deli = (char*)"/";
+    char* entry = NULL;
+    char  tag[5] = {0};
```

Comment:
Prefer `XMEMSET` to 0. Todd's told me to avoid this `{0}` construct in the past.